src/useQueue/__tests__/dom.ts

```diff
@@ -0,0 +1,50 @@
+import { act, renderHook } from '@testing-library/react-hooks/dom';
+import { useQueue } from '../..';
+
+describe('useQueue', () => {
+  it('should be defined', () => {
+    expect(useQueue).toBeDefined();
+  });
+
+  it('should render', () => {
+    const { result } = renderHook(() => useQueue());
+    expect(result.error).toBeUndefined();
+  });
+
+  it('should accept an initial value', () => {
+    const { result } = renderHook(() => useQueue([0, 1, 2, 3]));
+    expect(result.current.first).toBe(0);
+  });
+});
+
+describe('remove', () => {
```

Comment:
One describe block probably would suffice for the whole test file since there's only one test for each of the functions.

Reply:
This really is not a big thing. It's just a matter of grouping the tests logically (many tools also use the groupings to display the test results). To me it seemed odd that there's only one test per group and I haven't seen that anywhere else in the codebase.